### PR TITLE
feat(docs): Локализация docs/guide/recipes/custom-routing.md

### LINKED ISSUE #69: Локализация docs/guide/recipes/custom-routing.md
REF: https://nuxt-ru.vercel.app/docs/guide/recipes/custom-routing

### PR DESCRIPTION
**Pull request by mnenie**
### 🔗 Linked issue

resolves #69

### 📚 Description

Переведена страница docs/guide/recipes/custom-routing.md

### CODE REVIEW

File: docs/2.guide/4.recipes/1.custom-routing.md [hunk 1]
```diff
@@ -92,18 +92,18 @@ export default <RouterConfig> {
 }
 ```
 
-It is possible to add more router options files by adding files within the `pages:routerOptions` hook. Later items in the array override earlier ones.
+Можно добавить больше файлов опций маршрутизатора, добавив файлы в хук `pages:routerOptions` hook. Более поздние элементы массива переопределяют более ранние.
```
Comment (Hekikai) [inline docs/2.guide/4.recipes/1.custom-routing.md:95]:
```suggestion
Можно добавить больше файлов опций маршрутизатора, добавив файлы в хук `pages:routerOptions`. Более поздние элементы массива переопределяют более ранние.
```

File: docs/2.guide/4.recipes/1.custom-routing.md [hunk 2]
```diff
@@ -133,9 +133,9 @@ export default defineNuxtConfig({
 })
 ```
 
-### Hash Mode (SPA)
+### Режим хэширования (SPA)
 
-You can enable hash history in SPA mode using the `hashMode` [config](/docs/api/nuxt-config#router). In this mode, router uses a hash character (#) before the actual URL that is internally passed. When enabled, the **URL is never sent to the server** and **SSR is not supported**.
+Вы можете включить хэш-историю в SPA режиме с помощью `hashMode` [конфига](/docs/api/nuxt-config#router). В этом режиме маршрутизатор использует символ хэша (#) перед фактическим URL-адресом, который передается внутрь. Если этот режим включен, **URL никогда не отправляется на сервер** и **SSR не поддерживается**.
```
Comment (Hekikai) [inline docs/2.guide/4.recipes/1.custom-routing.md:138]:
```suggestion
Вы можете включить хэш-историю в SPA-режиме с помощью `hashMode` [конфига](/docs/api/nuxt-config#router). В этом режиме маршрутизатор использует символ хэша (#) перед фактическим URL-адресом, который передается внутрь. Если этот режим включен, **URL никогда не отправляется на сервер** и **SSR не поддерживается**.
```

**Review by Hekikai — CHANGES_REQUESTED** [review]

**Review by Ibochkarev — APPROVED** [review]

File: docs/2.guide/4.recipes/1.custom-routing.md [hunk 3]
```diff
@@ -92,18 +92,18 @@ export default <RouterConfig> {
 }
 ```
 
-It is possible to add more router options files by adding files within the `pages:routerOptions` hook. Later items in the array override earlier ones.
+Можно добавить больше файлов опций маршрутизатора, добавив файлы в хук `pages:routerOptions`. Более поздние элементы массива переопределяют более ранние.
```
Comment (alexeyvokin) [inline docs/2.guide/4.recipes/1.custom-routing.md:95]:
```suggestion
Можно добавить больше файлов опций маршрутизатора, добавив файлы в хуке `pages:routerOptions`. Более поздние элементы массива переопределяют более ранние.
```

File: docs/2.guide/4.recipes/1.custom-routing.md [hunk 4]
```diff
@@ -92,18 +92,18 @@ export default <RouterConfig> {
...
 
 ::alert
-Adding a router options file in this hook will switch on page-based routing, unless `optional` is set, in which case it will only apply when page-based routing is already enabled.
+Добавление файла опций маршрутизатора в этот хук включит постраничную навигацию, если только не задано значение `optional`, в этом случае она будет применяться только тогда, когда постраничная навигация уже включена.
```
Comment (alexeyvokin) [inline docs/2.guide/4.recipes/1.custom-routing.md:98]:
```suggestion
Добавление файла опций маршрутизатора в этом хуке включит постраничную навигацию, если только не задано значение `optional`, в этом случае она будет применяться только тогда, когда постраничная навигация уже включена.
```

Comment (alexeyvokin) [reply docs/2.guide/4.recipes/1.custom-routing.md:98]: хотя, конечно, можно так и так)

**Review by Hekikai — APPROVED** [review]

**Pull request merged**